plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1656,12 +1688,18 @@ public Optional<ConnectorOutputMetadata> finishCreateTable(ConnectorSession sess
             if (isRespectTableFormat(session)) {
                 verify(handle.getPartitionStorageFormat() == handle.getTableStorageFormat());
             }
+
+            List<Type> partitionTypes = handle.getPartitionedBy().stream()
+                    .map(columnTypes::get)
+                    .collect(toImmutableList());
             for (PartitionUpdate update : partitionUpdates) {
+                String partitionName = update.getName();
                 Partition partition = buildPartitionObject(session, table, update);
+                List<String> canonicalPartitionValues = canonicalizePartitionValues(partitionName, partition.getValues(), partitionTypes);
```

Comment:
Do we need that in `finishCreateTable` flow?

I think all partition names and value lists are canonical (because we created them).

Reply:
maybe we dont' need this, but then worth adding a code comment why we don't need this

Reply:
Good point.